jans-cli-tui/setup.py

```
@@ -6,10 +6,24 @@
 from setuptools.command.install import install
 from urllib.request import urlretrieve
+from urllib.error import URLError
+
+def _download_with_retry(url, dest, retries=5, delay=10):
+    for attempt in range(retries):
+        try:
+            urlretrieve(url, dest)
+            return
+        except (URLError, OSError) as e:
+            if attempt < retries - 1:
+                print("Download failed ({}), retrying in {}s...".format(e, delay))
```

Ruff (0.15.21)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)